README.md

```diff
@@ -143,6 +143,69 @@ if err != nil {
 	panic("add error handling")
 }
 ```
+#### Querying with parameters
+
+Users will sometimes want to use query parameters (Read more here - https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/queryparametersstatement?pivots=azuredataexplorer).
+In this case, the StatementBuilder and QueryParameters structs should be used.
+
+```go
+dt, _ := time.Parse(time.RFC3339Nano, "2020-03-04T14:05:01.3109965Z")
+// Query our database table "systemNodes" for the CollectionTimes and the NodeIds.
+statement := kql.NewStatementBuilder("systemNodes | where CollectionTime == time and NodeId == id")
+params :=  kql.NewStatementQueryParameters().AddDateTime("time", dt).AddInt("id", 1) 
+iter, err := client.Query(ctx, "database", statement, params)
+
+fmt.Println(fmt.Sprintf("%s\n%s", params.ToDeclarationString(), statement.String()))
+// Will print declare query_parameters(time:datetime, id:int)\nsystemNodes | where CollectionTime == time and NodeId == id 

```

Comment:
// Will print:
//  declare query_parameters(time:datetime, id:int);
//  systemNodes | where CollectionTime == time and NodeId == id
(In docs too)